src/ResourceManager/Compute/Commands.Compute/Models/PSVmssDiskEncryptionStatusContext.cs

```diff
@@ -24,11 +24,30 @@ class PSVmssDiskEncryptionStatusContext
         public string VmScaleSetName { get; set; }
         public AzureVmssDiskEncryptionExtensionPublicSettings EncryptionSettings { get; set; }
         public IList<VirtualMachineStatusCodeCount> EncryptionSummary { get; set; }
+        // EncryptionEnabled will be deprecated in the future and replaced with disk level granularity
+        // in a future version that accomodates breaking changes.  Currently true if any disk encrypted.
         public bool EncryptionEnabled { get; set; }
         public bool EncryptionExtensionInstalled { get; set; }
+        public IList<PSVmssDiskEncryptionInstanceStatusContext> Instances { get; set; }
```

Comment:
@ejarvi Could you update the related tests to check for the newly added properties?

Also we should call out the changes in the cmdlet (new properties and updated functionality) in the change log for the module.

Reply:
@praries880  updated tests and change log